README.md

```diff
@@ -84,6 +84,147 @@ To get started with contributing to the SurrealDB documentation, first install t
 bun i 
 ```
 
+## Contributing Lab content to SurrealDB Documentation
+
+## Overview
+Labs are a collection of community and official content that helps users learn and work with SurrealDB. This guide will help you contribute your own lab content to the documentation.
+
+## Prerequisites
+- [Bun](https://bun.sh/) installed on your system
+- A GitHub account
+- Basic knowledge of Markdown
+
+## Creating a new Lab
+
+### Using the Lab creation tool
+The easiest way to create a new lab is using the built-in tool:
+
+1. Clone the repository and install dependencies:
+
+```bash
+git clone https://github.com/surrealdb/docs.surrealdb.com.git
+cd docs.surrealdb.com
+bun install
+```
+
+2. Run the lab creation tool:
+
+```bash
+bun run make:lab
+```
+
+3. Follow the interactive prompts to provide:
+   - Lab name (required)
+   - URL (optional) - Link to your project/repository
+   - Category (required) - Choose from:
+     - CI/CD
+     - Demos
+     - Deployment Tools
+     - Development Tools
+     - Docker Images
+     - Integrations
+     - Libraries
+     - SDKs
+     - Templates
+     - Tutorials
+     - Videos
+   - Topics (optional) - Select one or more from:
+     - AI
+     - Data Management
+     - Embedding
+     - Security
+     - Examples
+     - Optimisation
+     - Beginner
+   - Author name (required)
+   - Author role (required for community content)
+
+### Manual creation
+Alternatively, you can create a lab manually by creating a new Markdown file in [`src/content/labs-items/`](https://github.com/surrealdb/docs.surrealdb.com/tree/main/src/content/labs-items) with the following structure:
+
+```markdown
+---
+title: "Your Lab Title"
+url: "https://your-project-url.com"  # Optional
+category: "Category Name"  # Must be one of the predefined categories
+topics:  # Optional
+  - Topic1
+  - Topic2
+author:
+  name: "Your Name"
+  role: "Your Role"
+  avatar: "your-name-slug"  # Will be automatically generated
+---
+
+Your lab content here...
+```
+
```

Comment:
We need to explain people will have to upload an avatar to `/src/assets/img/labs-avatars` with the same file name as the `author/avatar` property in the lab markdown